apps/astarte_appengine_api/lib/astarte_appengine_api_web/endpoint.ex

```diff
@@ -43,6 +38,9 @@ defmodule Astarte.AppEngine.APIWeb.Endpoint do
 
   plug Plug.RequestId
   plug PlugLoggerWithMeta
```

Comment:
I just realized that if we move `plug PlugLoggerWithMeta` _below_ `HealthPlug` and `MetricPlug`, we would also solve the spammy logs caused by metrics and health requests (the conn gets halted in `HealthPlug` and `MetricsPlug`, so it won't pass in `PlugLoggerWithMeta`).

@bettio opinion? I'm personally in favor, spammy logs makes it very hard to debug problems that happened e.g. during the weekend.

Reply:
I agree with this change